src/CommunityToolkit/Xamarin.CommunityToolkit/Views/LazyView/LazyView.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class LazyView<TView> : BaseLazyView where TView : View, new()
+	{
+		public override void LoadView()
+		{
+			View view = new TView { BindingContext = BindingContext };
```

Comment:
Any chance to create a method factory?

ex.:

```suggestion

		private readonly Func<TView> _factoryMethod;

		public LazyView ()
		{
		}

		public LazyView (Func<TView> factoryMethod)
		{
			_factoryMethod = factoryMethod;
		}

		public override void LoadView()
		{
			View view = (_factoryMethod?.Invoke() ?? new TView);
			view.BindingContext = BindingContext;
```

Reply:
@felipebaltazar not on this PR, the goal of this one is creating a really basic implementation of that `View`. After this gets merged please open an issue to add this feature. In meanwhile the `BaseFactoryMethod` is public so you can create your custom LazyView easily.